src/amp/components/elements/InteractiveBlockComponent.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { css } from 'emotion';
+import { ShowMoreButton } from '@root/src/amp/components/ShowMoreButton';
+
+const styles = css`
+	height: 100px;
+	width: 100%;
+	margin-top: 16px;
+	margin-bottom: 12px;
+`;
+
+const showMore = css`
+	&[overflow] {
+		position: absolute;
+		bottom: 0;
+		left: 0;
+		right: 0;
+	}
+`;
+
+export const InteractiveBlockComponent: React.SFC<{
+	url?: string;
```

Comment:
The majority of this is copied over from the interactive atom component that the interactive block element was already using